README.md

```diff
@@ -27,8 +27,8 @@ datasetinsights download \
 
 ```python3
 
-from datasetinsights.io.downloader import UnitySimulationDownloader,
-GCSDatasetDownloader, HTTPDatasetDownloader
+from datasetinsights.io.downloader import (UnitySimulationDownloader,
```

Comment:
I will separate these three downloader into three code blocks and explain briefly what each block is trying to achieve.